unlock-app/src/components/lock/Lock.js

```diff
@@ -73,6 +76,7 @@ Lock.propTypes = {
   purchaseKey: PropTypes.func.isRequired,
   config: UnlockPropTypes.configuration.isRequired,
   hideModal: PropTypes.func.isRequired,
+  updateLockPrice: PropTypes.func.isRequired,
```

Comment:
Should this even be required?

Reply:
Probably? I think you always want to have a function defined here because it will be invoked when users click on the `save` button.

Reply:
Yes!